test/Dialect/Tpp/tpp-bufferization.mlir

```diff
@@ -422,3 +381,130 @@ func.func @brgemm_in_loops(%arg0: tensor<4x16x32x32xf32>, %arg1: tensor<8x16x32x
 }
 
 // CHECK: tpp.brgemm ins(%{{.+}} : memref<16x32x32xf32, strided<[1024, 32, 1], offset: ?>>, %{{.+}} : memref<16x32x32xf32, strided<[1024, 32, 1], offset: ?>>, %{{.+}} : memref<32x32xf32, strided<[32, 1], offset: ?>>) outs(%{{.+}} : memref<32x32xf32, strided<[32, 1], offset: ?>>)
+
+// -----
+
+func.func @mlp_single_layer(%arg0: tensor<256x1024xf32>) -> tensor<256x1024xf32> {
```

Comment:
No bf16 tests? Removed above but not added elsewhere?

Reply:
We don't need to test multiple data type. The test I removed was wrong as it was in DPS style.